### DISCUSSION TITLE
Ways to filter certain Resource Types?

### DISCUSSION THREAD

**Discussion by Techsteps** [PrateekKumarSingh/AzViz#124] [General]
It works well, but I have like 100 Metric Alert Rules in my Resource groups, and I want just resource types revolving around the infrastructure itself (VMs, Storage, NICs...etc.) Is there a way to do this?

**Comment by zebak007** [comment]
From the code ....

.PARAMETER ExcludeTypes
 
Exclude Azure resources/providers from the visualization by passing them as an argument to the '-ExcludeTypes' parameter
 
Export-AzViz -ResourceGroup prateek -Show -ExcludeTypes "*workspace*", "Microsoft.Storage*" -Theme Neon
